src/test/java/org/openrewrite/java/testing/jmockit/JMockitVerificationsToMockitoTest.java

```diff
@@ -777,4 +775,122 @@ void test() {
           )
         );
     }
+
+    @Test
+    void whenWithRedundantThisModifier() {
+        //language=java
+        rewriteRun(
+          java(
+            """
+              import mockit.Verifications;
+              import mockit.Mocked;
+              import mockit.integration.junit5.JMockitExtension;
+              import org.junit.jupiter.api.extension.ExtendWith;
+              
+              @ExtendWith(JMockitExtension.class)
+              class MyTest {
+                  @Mocked
+                  Object myObject;
+              
+                  void test() {
+                      new Verifications() {{
+                          myObject.wait(this.anyLong, this.anyInt);
+                      }};
+                  }
+              }
+              """,
+            """
+              import org.junit.jupiter.api.extension.ExtendWith;
+              import org.mockito.Mock;
+              import org.mockito.junit.jupiter.MockitoExtension;
+              
+              import static org.mockito.Mockito.*;
+              
+              @ExtendWith(MockitoExtension.class)
+              class MyTest {
+                  @Mock
+                  Object myObject;
+              
+                  void test() {
+                      verify(myObject).wait(anyLong(), anyInt());
```

Comment:
Looks good. Could you add another test case which contains a mix of this and no this for the same method ? eg myOject.wait(this.anyString, anyInt)

Reply:
Also brilliant. I added a "mixture" and fortunately my code works well with it :)